File: Level1/수박수박수/no13.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(int n) {
    char* answer = (char*)malloc(sizeof(char) * n * 3 + 1);
    memset(answer, 0, sizeof(char) * n);
    
    for(int i = 1; i <= n; i++){
        if(i % 2 != 0) strcat(answer, "수");
        else strcat(answer, "박");
    }
    
    return answer;
}

File: Level1/수박수박수/no13.java
class Solution {
    public String solution(int n) {
        //StringBuffer answer = new StringBuffer("");
        StringBuilder answer = new StringBuilder("");
        
        for(int i = 1; i <= n; i++){
            answer = i % 2 != 0 ? answer.append("수") : answer.append("박");
        }
        
        return answer.toString();
        
        /*

        String answer = "";
        
        for(int i = 1; i <= n; i++){
            if(i % 2 != 0) answer += "수";
            else answer += "박";
        }
        
        return answer;
        
        */
    }
}